scripts/src/main/resources/scripts/command/docker

```diff
@@ -162,17 +216,19 @@ function doInstallVpnkit() {
   #write a powershell-script
   #go to an actual windows path for the powershell command by using pushd to be able to get back easily with popd
   #execute command to start wsl-vpnkit
+  #redirect messages that won't be helpful for user
   # shellcheck disable=SC1083,SC2129,SC2016
   {
-    echo 'mv -Force wsl-vpnkit.tar.gz $env:USERPROFILE\wsl-vpnkit.tar.gz' 
-    echo 'pushd $env:USERPROFILE'
-    echo 'wsl --import wsl-vpnkit $env:USERPROFILE\wsl-vpnkit wsl-vpnkit.tar.gz --version 2'
-    echo 'popd'
-    echo 'wsl.exe -d wsl-vpnkit service wsl-vpnkit start' 
+    echo 'mv -Force wsl-vpnkit.tar.gz $env:USERPROFILE\wsl-vpnkit.tar.gz | Out-Null' 
+    echo 'pushd $env:USERPROFILE | Out-Null'
+    echo 'wsl --import wsl-vpnkit $env:USERPROFILE\wsl-vpnkit wsl-vpnkit.tar.gz --version 2 | Out-Null'
+    echo 'popd | Out-Null'
+    echo 'wsl.exe -d wsl-vpnkit service wsl-vpnkit start | Out-Null' 
   } > vpnkit-setup.ps1
 
   #execute the powershell-script 
-  powershell.exe -Command ".\vpnkit-setup.ps1"
+  # shellcheck disable=2091
+  $(powershell.exe -Command ".\vpnkit-setup.ps1")
```

Comment:
I do not get the rationale for this change. `$()` syntax is used to get the output of an execution.
As you are not using the output (e.g. as in `variable="$(command)"`) you can simply omit this syntax, but why did you add it then?